_schemaV2.graphql

```diff
@@ -8108,6 +8095,37 @@ type Tag implements Node {
   slug: ID!
 }
 
+type TargetSupply {
+  microfunnel: [TargetSupplyMicrofunnelItem]
+}
+
+type TargetSupplyMicrofunnelItem {
+  artist: Artist
+
+  # A list of recently sold artworks.
+  artworksConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+
+    # Randomize the order of artworks for display purposes.
+    randomize: Boolean
```

Comment:
Noting again that `size` arg was removed when I made the type shared between the `artist` field and the root field.